### ISSUE TITLE
tetrahedral SBP operators

### ISSUE THREAD

**Issue by jlchan** [jlchan/StartUpDG.jl#13]

**Comment by DanielDoehring** [comment]
Hi Jesse, is there any update on this (maybe also @PhilBaa is working on this?)

**Comment by jlchan** [comment]
No, but the construction should be immediate once I get the points and weights. I think Zelalem's recent paper has these but I haven't extracted them yet.

**Comment by DanielDoehring** [comment]
Are you referring to the 2024 paper? https://link.springer.com/article/10.1007/s10915-024-02656-5 This sounds like it is the one we are looking for - or are you referring to https://www.sciencedirect.com/science/article/pii/S0377042725002961 ?

**Comment by jlchan** [comment]
Both, actually - though I would imagine it makes more sense to use https://www.sciencedirect.com/science/article/pii/S0377042725002961

**Comment by DanielDoehring** [comment]
Would it be possible to query what we need from Zela's package ( https://github.com/[OptimalDesignLab/SummationByParts.jl](https://github.com/OptimalDesignLab/SummationByParts.jl ) ?

See e.g. 

https://www.optimaldesignlab.com/SummationByParts.jl/stable/reference/#SummationByParts.SparseTetSBP

**Comment by jlchan** [comment]
Yes, I think so - that has all the info we would need. I can take a stab at
adding those this weekend, or if you'd like to submit a PR I'm happy to
provide feedback?

**Comment by DanielDoehring** [comment]
Hm I am not sure when I have the time to do this - I guess whoever is first can start :) But please do not feel rushed - for me this is really now a hobby/curiosity :)
